projekt-03-kontakty-main/projekt-03-kontakty-main/style.css

```diff
@@ -8,3 +8,44 @@ body {
 }
 
 /* ---------------------------------------------------- */
+
+.container {
+	display: inline-flexbox;

```

Comment:
Tady pozor: `inline-flexbox` není validní honota vlastnosti display, správně je `inline-flex`.
`inline-flex` by tady klidně mohl být, ale je potřeba si uvědomit, že nám narozdíl od flex určuje to, jak se bude element chovat zvenku vůči ostatním elementům (obdobně jako `block` a `inline-block`) a na uspořádání itemů uvnitř to nic nemění (to je stejné pro` flex` i `inline-flex`). Můžeš kouknout na tenhle fiddle, kde je vidět rozdíl: http://jsfiddle.net/mgr0en3q/1/

Reply:
Takže abych to shrnula, nijak nám to tu neškodí, ale zároveň tady tu funkcionalitu inline-flexu nepotřebujeme a flex by stačil 🙂